Mathlib/LinearAlgebra/LinearIndependent.lean

```diff
@@ -119,11 +124,65 @@ def delabLinearIndependent : Delab :=
       withNaryArg 0 do return (← read).optionsPerPos.setBool (← getPos) `pp.analysis.namedArg true
     withTheReader Context ({· with optionsPerPos}) delab
 
-variable {R} {v}
+variable {R v}
+
+theorem linearIndependent_iff_injective_linearCombination :
+    LinearIndependent R v ↔ Function.Injective (Finsupp.linearCombination R v) := Iff.rfl
+
+alias ⟨LinearIndependent.injective_linearCombination, _⟩ :=
```

Comment:
Don't we usually put "injective" at the end as an exception to the naming convention?

Reply:
Because `LinearIndependent.injective_linearCombination` is an existing result in mathlib, as well as `linearIndependent_iff_injective_linearCombination`.

Reply:
Yes, all these declarations are just moved.